docs/basics/getting-started.md

```diff
@@ -1,210 +1,159 @@
----
-id: getting-started
-title: Getting Started
-sidebar_position: 0
----
+# Getting Started
 
 ## Introduction
 
-Cross-seeding is the term used for downloading a torrent from one tracker and then using that data to seed across different trackers.
-Generally, this is not against the rules, but you should always check and abide by your trackers' rules.
-
-Since you already have the torrent's files, this results in zero download and instant seeding. This is a great way to build ratio
-(especially on your new trackers) and contribute to the torrenting community.
+Cross-seeding is the practice of downloading a torrent from one tracker and
+using that data to seed across other trackers. This results in minimal downloads
+and instant seeding, making it a great way to build ratio and contribute to the
+community.
 
 :::info
-This software is designed to work mainly with private trackers. If you are attempting to `cross-seed` on public trackers, please
-see our [FAQ entry](./faq-troubleshooting.md#does-cross-seed-work-on-public-trackers) for details before proceeding.
-:::
 
-:::note
-The primary documentation for v6 of `cross-seed` during pre-release is located [here](../v6-migration.md) in the v6 migration guide.
-Please refer to this guide if you are using v6 and have questions about the changes. Set up and basic configurations described elsewhere
-in this documentation that are not present in the migration guide are still valid.
+This software is designed primarily for **private trackers**. If you plan on
+cross-seeding on public trackers,
+[please see our FAQ](./faq-troubleshooting.md#does-cross-seed-work-on-public-trackers)
+for more details.
 
-If you have any questions please feel free to contact us on Discord or the Discussions section of GitHub. There are links located as
-icons in the top right of every page.
 :::
 
-:::danger
-Certain `cross-seed` configurations can result in downloading an excessive number of torrent files in a short amount of time.
-Trackers may, as a result, forbid this or require specific settings to minimize this behavior as much as possible.
+---
 
-[See this FAQ post for details](./faq-troubleshooting.md#my-tracker-is-mad-at-me-for-snatching-too-many-torrent-files)
-:::
+## Setting Up the Daemon
+
+The most efficient way to use `cross-seed` is to run it in **daemon mode**,
+which continuously monitors for new downloads and cross-seeds them
+automatically. Here’s how to get everything up and running.
 
-<hr/>
-Now we will install `cross-seed` and run our first search.
+### 1. Install `cross-seed`
 
-## Installation
+You can install `cross-seed` using one of the following methods:
 
-### with `npm` (recommended if you're not already using Docker)
+#### With `npm` (recommended if you're not using Docker)
 
-Requires [node](https://nodejs.org/en/download/) 16 or greater.
+You’ll need **Node.js 20** or greater.
 
 ```shell
 npm install -g cross-seed
 cross-seed --version
 ```
 
-### with Docker
+#### With Docker
```

Comment:
I think alot of people will be looking for a compose template/example as an installation method. While this is technically installing it with docker (placing the image on their computer to run later) you seem to have completely removed our compose example from this page entirely. 

I think linking it here would be a logical place to give them the compose example, if not directly including a template in this section.

I can't recall running across any significant number of people who used docker strictly via `docker run` commands as their method of execution, and that seems to be what is defaulted to here.

Reply:
good idea, that's what I would want too. [`29a6511` (#58)](https://github.com/cross-seed/cross-seed.org/pull/58/commits/29a65113710f778ec18d632f7930d11eabef153d)